tools/wptrunner/wptrunner/executors/executorselenium.py

```diff
@@ -336,7 +334,11 @@ def is_alive(self):
     def do_test(self, test):
         self.logger.info("Test requires OS-level window focus")
 
-        self.protocol.webdriver.set_window_size(600, 600)
+        width_offset, height_offset = self.protocol.webdriver.execute_script(
+            """return [window.outerWidth - window.innerWidth,
+                       window.outerHeight - window.innerHeight];"""
+        )
+        self.protocol.webdriver.set_window_size(800 + width_offset, 600 + height_offset)
```

Comment:
This needs changed similarly in executorwebdriver; we should probably just rebase this on top of #14922 so we make everything correct everywhere then change the viewport size.